features/measurements/impl/src/test/java/org/opennms/netmgt/measurements/filters/impl/LoadData.java

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
```

Comment:
copyright headers can be updated to new header that is independent of year. 
see copyright header on TrendLine.java

Reply:
My question is, should I make the required changes?